cfpb/design-stories
# [Components] Notification / Icon

Closes [#373](https://github.cfpb.gov/reg-tech/sbl-data-collection/issues/373)
 
A bit of scope creep here but I wanted the implementation of icons to be reusable.  

## Testing

1. Pull down branch.
2. `yarn && yarn start`
3. View the [icon](http://localhost:6006/?path=/docs/components-icon--icon) and [notification](http://localhost:6006/?path=/docs/components-notification--information) stories
4. They should look similar to the DS [notifications](https://cfpb.github.io/design-system/components/notifications) and [icons](https://cfpb.github.io/design-system/foundation/iconography)


## Requesting feedback on
- File structure
  - Should we split out `css` into a `/styles` folder?
  - Should we move `stories` to the `/stories` folder?
  - Should we keep everything in `/components` so that its easily accessible without navigating 50 different folders?
- Icon implementation
  - I tried for 3 days to get [vite-plugin-svgr](https://github.com/pd4d10/vite-plugin-svgr) to work so that we could import SVG similar to how it's done in CRA...no dice.  So I implemented the dynamic import + `dangerouslySetInnerHTML` approach because it was the only way I found that allowed us to change the `fill` color of SVG elements.
